config/logstash.yml

```diff
@@ -241,6 +244,9 @@
 #xpack.management.elasticsearch.username: logstash_admin_user
 #xpack.management.elasticsearch.password: password
 #xpack.management.elasticsearch.hosts: ["https://es1:9200", "https://es2:9200"]
+# an alternative to hosts + username/password settings is to use cloud_id/cloud_auth
+#xpack.management.elasticsearch.cloud_id: management:xxxxxxxxxx
```

Comment:
Maybe use something like `management_cluster_id:xxxxx` rather than `management:xxxxx` as using `management` here could be ambiguous